src/routes/safe/container/index.jsx

```diff
@@ -48,6 +49,11 @@ class SafeView extends React.Component<Props, State> {
     }, TIMEOUT)
   }
 
+
+  shouldComponentUpdate(nextProps: Readonly<P>, nextState: Readonly<S>, nextContext: any): boolean {
+    return !isEqual(nextProps, this.props)
```

Comment:
Was there a particular reason for this check? I do know that there is a problem with re-renders and we should do some optimization in the future.

Also afaik there are faster options than lodash's `isEqual`, so if there's a particular reason for this check, maybe we can swtch to one of these:
https://stackoverflow.com/questions/52949795/why-should-i-avoid-to-use-isequal-in-shouldcomponentupdate

Reply:
Now it makes no sense, I just removed it, before, this was needed because the safe pages was always re-rendering causing that the fetchTransactions action executes all the time